roles/qemu_setup/tasks/main.yml

```diff
@@ -65,6 +65,8 @@
     daemon_reload: true
     enabled: true
   become: true
+  loop:
```

Comment:
I'm not sure I understand why you added the loop here

Reply:
I guess there were some other services he wanted to restart apart from libvirtd (maybe something to do with the new bridged network?) and also forgot to replace libvirtd with "{{ item }}"?